### PR TITLE
[Android] Update rich text block

### PR DESCRIPTION
**Pull request by almedina-ms**
Updates the RichTextBlock to remove the paragraphs and keep the same behaviour as the one in UWP

### CODE REVIEW

**Comment by RebeccaAnne** [comment]
Can you set this pull request to merge into my branch instead of master? I think we should probably consolidate everything there and then once we're all in place we can go to master.

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/RichTextBlockRenderer.java [hunk 1]
```diff
@@ -188,49 +180,19 @@ else if ((richTextBlock = RichTextBlock.dynamic_cast(baseCardElement)) == null)
...
-            if ((i != 0) && (inlines.size() != 0))
-            {
-                textView.append(System.getProperty("line.separator"));
-            }
-
-            SpannableStringBuilder convertedString = buildSpannableParagraph(renderedCard, inlines, cardActionHandler, hostConfig, renderArgs);
-
-            textView.append(convertedString);
-        }
+        SpannableStringBuilder convertedString = buildSpannableParagraph(renderedCard, inlines, cardActionHandler, hostConfig, renderArgs);
```
Comment (RebeccaAnne) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/RichTextBlockRenderer.java]: >buildSpannableParagraph [](start = 49, length = 23)

Does this change include removing markdown support for rich text blocks?

**Comment by RebeccaAnne** [comment]
It also makes the code review easier so we can see which changes are part of which PRs

---
In reply to: [483372991](https://github.com/Microsoft/AdaptiveCards/pull/2688#issuecomment-483372991) [](ancestors = 483372991)

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/RichTextBlockRenderer.java (hunk 1 above)
Comment (almedina-ms) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/RichTextBlockRenderer.java]: now it does :D

---
In reply to: [275498255](https://github.com/Microsoft/AdaptiveCards/pull/2688#discussion_r275498255) [](ancestors = 275498255)

**Pull request merged**